readme.md

```diff
@@ -51,25 +51,25 @@ Accelerando is an excellent exploration of Posthumanism. It's my go to recommend
 
 It's also worth mentioning that [the ebook is available for free in a variety of formats on Stross's website](https://www.antipope.org/charlie/blog-static/fiction/accelerando/accelerando-intro.html). - [@AlexKeyes](https://github.com/alex-keyes)
 
-> The Singularity. It is the era of the posthuman. Artificial intelligences have surpassed the limits of human intellect. Biotechnological beings have rendered people all but extinct. Molecular nanotechnology runs rampant, replicating and reprogramming at will. Contact with extraterrestrial life grows more imminent with each new day.
->
-> Struggling to survive and thrive in this accelerated world are three generations of the Macx clan: Manfred, an entrepreneur dealing in intelligence amplification technology whose mind is divided between his physical environment and the Internet; his daughter, Amber, on the run from her domineering mother, seeking her fortune in the outer system as an indentured astronaut; and Sirhan, Amber's son, who finds his destiny linked to the fate of all of humanity.
->
-> For something is systemically dismantling the nine planets of the solar system. Something beyond human comprehension. Something that has no use for biological life in any form.
+<details>
+  <summary>Description</summary>
 
-##### [Adulthood Rites](https://www.goodreads.com/book/show/19105727-adulthood-rites) (1988) [4.2]
```

Comment:
Why did you delete this one? Can you double-check all of these?

Reply:
They were out of sequence (Level 4 headings mixed in with Level 3 headings) and missing the standard info (author etc). There's an entry for the trilogy (Xenogenesis) elsewhere with that info, where they fit in. I figured that was the most correct way to handle them?